e2e/autoscript-apps/src/authn-oauth/autoscript.ts

```diff
@@ -88,6 +88,7 @@ function autoscript() {
         } else {
           throw new Error('Session_Error');
         }
+        console.log(tokens.rawResponse);
```

Comment:
Adding a log of the raw response, so the test suite can pick it up and ensure the original response has the correct values.

Reply:
Should probably use the Logger class?